.github/renovate.json

```diff
@@ -38,9 +40,20 @@
       "groupName": ["DevDependencies (non-major)"],
       "automerge": true
     },
+    {
+      "extends": ["packages:test"],
+      "matchUpdateTypes": ["patch", "minor"],
+      "matchCurrentVersion": "!/^0/",
+      "groupName": "Test packages (non-major)",
+      "automerge": true
+    },
+    {
+      "extends": ["packages:test"],
+      "matchUpdateTypes": ["major"],
+      "groupName": "Test packages"
```

Comment:
```suggestion
      "groupName": "Test packages (major)"
```

Reply:
don't need to add that.  Renovate will do it for you. See https://github.com/janus-idp/backstage-plugins/blob/main/.github/renovate.json#L93-L99 and test PRs in dependency dashboard: https://github.com/janus-idp/backstage-plugins/issues/19